### ISSUE TITLE
Up and Down for Blinds

### ISSUE THREAD

**Issue by hamboy75** [hardillb/node-red-alexa-home-skill-web#145]
Hi, i have 21 Blinds with tasmota in my house. I have conected them as switch, but then then voice commands are Turn On and Turn off (well in spanish Encience and Apaga instead of up/down, Sube and Baja in spanish ) that is the proper way to use them.

Another thing, would be nice to have device with 2 switches. Actually i'm using an "Up kitchen and a Down Kitchen device", it is a bit unnatural say "turn on Up Kitchen" and "Turn on Down Kitchen".

Would be nice to have only a "kitchen" device for example and tell alexa to "up kitchen", "down kitchen" or "Stop kitchen"
Is this possible already? 

Best regards.

**Comment by Bra1nK** [comment]
node-red-contrib-virtual-smart-home does blinds, node-red-alexa-home-skill-web is in maintenance mode only and further developement is unlikely

**Comment by hardillb** [comment]
No, none of the devices supported by the level of API the skill is built on have up/down actions.
